### DISCUSSION TITLE
[Fixed] Not able to run the ui.shadcn.com website

### DISCUSSION THREAD

**Discussion by ShrinidhiUpadhyaya** [shadcn-ui/ui#3148] [General]
Hi All,

i am facing some issues while trying to run **'ui.shadcn.com'** locally

I have installed all the dependencies using `pnpm install`

When i try to run the ui.shadcn.com website, `pnpm --filter=www dev`, i get the following error

![image](https://github.com/shadcn-ui/ui/assets/8769408/98734824-c581-4094-b8cc-b3d1ab921c1c)

I tried to install the opentelemetry dependency using `pnpm i @opentelemetry/exporter-trace-otlp-grpc --force -w`, but i get the following error

![image](https://github.com/shadcn-ui/ui/assets/8769408/ebd79c3b-430f-43a3-8d02-113a95c7f757)

Could someone help me resolve this?

**Comment by shadcn** [comment]
Which node version are you on?

**Comment by ShrinidhiUpadhyaya** [comment]
I fixed it. The issue was i had "-" (hyphen) in my root folder where the code was.
